src/main/java/frc/robot/RobotContainer.java

```diff
@@ -60,6 +60,9 @@ public RobotContainer() {
     m_intake = new Intake();
     m_turret = new Turret();
 
+    m_driveJoystick = new Joystick(RobotMap.JOYSTICK.DRIVE_JOYSTICK);
+    m_operatorJoystick = new Joystick(RobotMap.JOYSTICK.OPERATOR_JOYSTICK);
+
```

Comment:
Nitpick: Can you please put these before the subsystems, so that we match the same order as in the declarations (Line 38-48).